lms/static/scripts/postmessage_json_rpc/server/methods.js

```diff
@@ -14,3 +14,24 @@ export function requestConfig() {
   const clientConfigObj = JSON.parse(configEl.textContent).hypothesisClient;
   return clientConfigObj;
 }
+
+/**
+ * Return the groups for the Hypothesis client to show.
+ */
+export async function requestGroups() {
+  const configObj = JSON.parse(
+    document.querySelector('.js-config').textContent
+  );
+
+  function sleep(ms) {
+    return new Promise(resolve => setTimeout(resolve, ms));
+  }
+
+  if (configObj.dev === true) {
+    // Artifically sleep to simulate how long this will take when it needs to
+    // send real API requests to get the groups.
+    await sleep(1900);
```

Comment:
Might be a better test if this was quick or slow randomly, in case we end up relying on the slow behavior.

Reply:
Done. Also increased the artificial delay so that it's longer than the client's current RPC response timeout of two seconds (a timeout that is going to have to be extended or removed, for sections)